test/langtools/tools/jdeps/listdeps/ListModuleDeps.java

```diff
@@ -95,6 +95,7 @@ public void compileAll() throws Exception {
                                 "java.base/jdk.internal.javac",
                                 "java.base/jdk.internal.jmod",
                                 "java.base/jdk.internal.misc",
+                                "java.base/jdk.internal.vm.annotation",
```

Comment:
This looks a bit suspicious to me - why the new entry? Do we know why it was necessary? (I would expect that any change here would be caused by a change in dependencies among modules, but I don't seem to see that change.)